content/guide/07-element-directives.md

```diff
@@ -517,3 +517,96 @@ Use actions for things like:
 	}
 }
 ```
+
+#### Classes
```

Comment:
This should be `### Classes`, so that it creates a navigation entry, instead of being a sub-sub-section under 'Actions'.